php-checks/src/test/resources/checks/security/CORSPolicyCheck.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace core;
+
+header("Access-Control-Allow-Origin: *"); // Noncompliant
+header("Access-Control-Allow-Origin:* "); // Noncompliant
+Header("access-control-allow-origin:     *   "); // Noncompliant
+header(code:42, replace:true, header:"Access-Control-Allow-Origin: *"); // Noncompliant
+
+header("Access-Control-Allow-Origin: " + $_SERVER['HTTP_ORIGIN']); // Compliant - FN
+header("Access-Control-Allow-Origin: *foo*");
+header("Access-Control-Allow-Origin", "*");
+header("Other-Property: *"); // Compliant
+header(); // Compliant
+
+class A {
+    function foo() {
+        header("Access-Control-Allow-Origin: *"); // Noncompliant
+        header("Access-Control-Allow-Origin: foo"); // Compliant
+    }
+}
+
+namespace laravel;
+use Illuminate\Http\Response;
+
+$response = new Response();
+$response->header('Access-Control-Allow-Origin', "*"); // Noncompliant
+response()->header('Access-Control-Allow-Origin', "*"); // Noncompliant
+$response->HEADER('  access-control-allow-origin  ', "  *  "); // Noncompliant
+$response->header(values:"*", key:'Access-Control-Allow-Origin'); // Noncompliant
+
+$response->header('Access-Control-Allow-Origin', "foo"); // Compliant
+response()->header('Access-Control-Allow-Origin', "foo"); // Compliant
+$response->header('Other-Property', "*"); // Compliant
+
+
+
+class A {
+    function foo() {
+        $response = new Response();
+        $response->header('Access-Control-Allow-Origin', "*"); // Noncompliant
+        $response->header('Access-Control-Allow-Origin', "foo"); // Compliant
+    }
+}
+
+namespace symfony;
+use Symfony\Component\HttpFoundation\Response;
```

Comment:
I'm not sure that our symbol table handles correctly the fact that we have 2 `use` statements in the same file with different namespaces and no alias.